voc/python/ast.py

```diff
@@ -880,6 +896,12 @@ def visit_Break(self, node):
             if loop.end_op is None:
                 break
         self.context.add_opcodes(
+            java.New('org/python/types/Bool'),
+            JavaOpcodes.ICONST_0(),
+            java.Init('org/python/types/Bool', 'Z'),
+            ASTORE_name('#for-orelse-%x' % id(loop))
+        )
```

Comment:
Last update from @fedeisas renames the flag to '#loop-orelse-%x' and includes support to the while/else clause.